### ISSUE TITLE
Developing Microservices on a Budget using AWS Lambda and Amazon SQS , Yes It is on AWS, Yes I can Pre-record the content, Yes I can provide the slides

### ISSUE THREAD

**Issue by mrshukla1805** [richarvey/aws-community-radio#1]
**Is your talk about technology on AWS?**
A clear and concise description of what the talk is about. 

**How long is your talk?**
Please try and keep talks to 25 mins maximum

**Can you pre-record the content?**
When third parties do a presentation we like to have pre-recorded content. This free's up the presenter to answer questions in the chat and then take some live questions later.

**Can you provide the slides before he talk?**
We like to let participants download the slides after the talk but we need to prepare this.

**Additional context**
Add any other context or screenshots about the talk proposed here.

**Comment by richarvey** [comment]
Thanks, @mrshukla1805, lets get this going. Are you looking to do this in one video or over a couple of videos? I'm happy with either way.

My final question is would you like to do a pre-interview before the show so I can introduce you to everyone.

**Comment by richarvey** [comment]
Just following up for interest in this show. Would you like to record and send over?

**Comment by mrshukla1805** [comment]
Yes I will record and send I would need a week to work on the project will get back to you as an when I have the recording ready!!! Thank you ;)

**Comment by richarvey** [comment]
Perfect, I'll schedule it in and work out the time for broadcast for you. Let know when you are ready and thank you!
